migrations/2022-01-29-101955_create_taglines/up.sql

```diff
@@ -0,0 +1,4 @@
+create table tagline (
```

Comment:
Add a foreign key reference to `local_site_id`, check the `local_site_rate_limit` table creation for an example of this.

Reply:
Also add published and updated columns. Copy paste from existing table creations or you will miss things like that.

Reply:
Ah thank you. I was thinking of that as I was working on this. I will add that in.

Reply:
Thanks, I was thinking about adding that in as I was deving but wasn't in the original issue. Will add it now.

Reply:
Actually is the local_site_id really necessary here? This data doesnt get federated, so you will never be able to read or write taglines for any other site than the local one. Same goes for rate limits now that i think about it.

Reply:
Yes, mainly because its not good to have unconnected, "dangling" data. We can't anticipate the data transformations and joins we might need. Plus both are logically local site settings.